src/ensembl/src/shared/error-screen/ErrorScreen.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { HomeLink, ReleaseVersion, Copyright } from 'src/header/Header';
+
+import styles from './ErrorScreen.scss';
+import headerStyles from 'src/header/Header.scss';
+
+import generalErrorImage1 from './images/general-error-1.jpg';
+import generalErrorImage2 from './images/general-error-2.jpg';
+import generalErrorImage3 from './images/general-error-3.jpg';
+
+export const NewTechError = () => {
+  return (
+    <div className={styles.newTechErrorWrapper}>
+      <div className={styles.newTechMessage}>
+        Sorry, but it seems your browser can’t display the new site
+      </div>
+      <a href="https://www.ensembl.org" className={styles.errorLinkButton}>
+        Go to Ensembl classic
+      </a>
+    </div>
+  );
+};
+
+export const GeneralErrorScreen = () => (
```

Comment:
Won't it be better to have these two components in separate files instead in the same one?

Reply:
No idea 😄 We don't seem to have a convention for exporting closely related things. We may use an `index.ts` file in the error-screen folder, which will import and export the `GeneralErrorScreen` component and the `NewTechError` component from sibling files sitting in the same folder, if you prefer that approach.

Reply:
That sounds good to me. Will make our import statements shorter 😄

Reply:
Ok, moved components to separate files and expose them via index.ts